components/productions/create/event.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <b-card class="card-event">
+    <section class="header">
+      <div>
+        <p>{{ type }}</p>
+        <h2>{{ title }}</h2>
+        <p>{{ period }}</p>
+      </div>
+      <img :src="imageUrl" :alt="title" class="image" />
+    </section>
+    <b-card-text>{{ description }}</b-card-text>
+  </b-card>
+</template>
+
+<script>
+  export default {
+    props: {
+      id: {
+        type: String,
+        default: '',
+      },
+      type: {
+        type: String,
+        default: '',
+      },
+      title: {
+        type: String,
+        default: '',
+      },
+      startDate: {
+        type: String,
+        default: '',
+      },
+      endDate: {
+        type: String,
+        default: '',
+      },
+      description: {
+        type: String,
+        default: '',
+      },
+      imageUrl: {
+        type: String,
+        default: '',
+      },
+    },
+    data() {
+      return {
+        period: '',
+      };
+    },
+    computed: {
+      locale() {
+        return this.$i18n.locale;
+      },
+    },
+    async created() {
+      this.period = await this.getPeriod();
+    },
+    methods: {
+      async getPeriod() {
+        return await this.$api.events.getCalendarSummary({
+          id: this.id,
+          locale: this.locale,
+        });
+      },
+    },
```

Comment:
Can't we just call the API helper directly? this just adds another layer of abstraction that isn't needed